tests/benchmarks/test_benchmark_td3.py

```diff
@@ -0,0 +1,205 @@
+"""
```

Comment:
please make sure this conforms to the format used by rlworkgroup/benchmarks

Reply:
Fixed.